### PR TITLE
give higher priority for modules in frontend

### PR DESCRIPTION
**Pull request by ramitos**

### CODE REVIEW

**Comment by ramitos** [comment]
only merge after #240

File: frontend/src/components/new-project/billing.js [hunk 1]
```diff
@@ -59,8 +59,8 @@ const Buttons = styled.div`
 `;
 
 const LeftButton = styled(Button)`
-  margin-right: ${remcalc(6)} !important;
-`; // But why oh why do I need to use !important :'(
```
Comment (alex-windett) [inline frontend/src/components/new-project/billing.js]: Great comment

Comment (ramitos) [inline frontend/src/components/new-project/billing.js]: 😂 it's a @juditgreskovits work of art

Comment (juditgreskovits) [inline frontend/src/components/new-project/billing.js]: ;)

**Pull request merged**